packages/messaging/src/Message.ts

```diff
@@ -156,9 +156,12 @@ export function verifyMessageBody(body: MessageBody): void {
       break
     }
     case 'submit-credential': {
-      body.content.forEach((credential) =>
+      body.content.forEach((credential) => {
```

Comment:
I would at least call the parameter for the forEach `presentation`, no?

Reply:
right